src/components/ReportActionItem/MoneyReportView.tsx

```diff
@@ -53,8 +55,8 @@ function MoneyReportView({report, policyReportFields, shouldShowHorizontalRule}:
         StyleUtils.getColorStyle(theme.textSupporting),
     ];
 
-    const sortedPolicyReportFields = useMemo(
-        () => policyReportFields.sort(({orderWeight: firstOrderWeight}, {orderWeight: secondOrderWeight}) => firstOrderWeight - secondOrderWeight),
+    const sortedPolicyReportFields = useMemo<PolicyReportField[]>(
+        (): PolicyReportField[] => policyReportFields.sort(({orderWeight: firstOrderWeight}, {orderWeight: secondOrderWeight}) => firstOrderWeight - secondOrderWeight),
```

Comment:
> `policyReportFields.sort`

Please use `toSorted`!

I know that you just touched this part, but you're aware of what this code does, right? We're mutating the array inside the `useMemo` builder.